tests/test_hygiene.py

```
@@ -194,6 +196,81 @@ def test_audit_scans_memories_table(self, temp_db):
+            min_score=0.3,
+            scan_all=True,
+            batch_size=2,
+        )
+
+        assert paged.total_scanned == 1
+        assert full.total_scanned == 3
+        assert len(full.candidates) == 3
+
+    def test_noise_summary_is_pii_safe(self, temp_db):
+        db_path, beam = temp_db
+        secret_content = "password = hunter2supersecret"  # nosec - test fixture
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to: "secret_content"

(S105)